src/vector.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Vector utility functions shared within the Genotet.
+ */
+
+'use strict';
+
+/** @const */
+genotet.vector = {};
+
+/**
+ * Calculates whether the point is in the polygon.
+ * @param {!Array<number>} point The point.
+ * @param {!Array<Array<number>>} vertices The vertices of the polygon.
+ * @return {boolean} The result.
```

Comment:
The result is redundant in this case. If return value description is already in the main description above, then it's fine to omit.